rest_api/test/test_rest_api.py

```diff
@@ -215,6 +215,23 @@ def test_query_with_invalid_filter(populated_client: TestClient):
     assert len(response_json["answers"]) == 0
 
 
+def test_query_with_no_documents():
```

Comment:
I'd say let's rename to `test_query_with_no_documents_and_answers()` because this test checks both.